setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import io
+
+from setuptools import setup
+
+# README into long description
+with io.open('README.rst', encoding='utf-8') as readme_file:
```

Comment:
Although this is technically for the Python 3 docs, using `io.open` over `open` is pretty cheap for us, especially considering this page contains almost no Python code. :)